src/validation.cpp

```diff
@@ -34,7 +35,6 @@
 #include <timedata.h>
 #include <tinyformat.h>
 #include <txdb.h>
-#include <txmempool.h>
```

Comment:
style-nit: Why shuffle the includes for no reason?

Reply:
Oops, will fix.